src/mongocxx/test/collection.cpp

```diff
@@ -678,9 +713,9 @@ TEST_CASE("CRUD functionality", "[driver::collection]") {
         bulk.append(model::insert_one{std::move(doc3)});
         bulk.append(model::insert_one{std::move(doc4)});
 
-        distinct_coll.bulk_write(bulk);
+        coll.bulk_write(bulk);
```

Comment:
Why are we using a different collection?  The commit message "minor cleanup" doesn't explain enough.